.gitignore

```diff
@@ -168,3 +168,6 @@ dmypy.json
 # Pyre type checker
 .pyre/
 .vscode
+
+# Mac OS
+.DS_Store
```

Comment:
Pedantic I know, but add OS and personal choice tool-specific stuff like IDE cruft to your global gitignore, so you don't have to mess with it for every project: see `git help gitignore`, but TL;DR it's `~/.config/git/ignore`